src/validations/classes.js

```diff
@@ -0,0 +1,24 @@
+const Joi = require('joi');
+
+const validateCreation = (res, req, next) => {
```

Comment:
This const could be called validateClassCreation to be consistent with the code of the other resources of the app